eng/pipelines/runtime.yml

```diff
@@ -368,6 +368,13 @@ jobs:
     - Windows_NT_arm
     - Windows_NT_arm64
     - Windows_NT_x86
+    # Generic test build always needs Release libraries.
+    - ${{ and(
```

Comment:
This condition is not correct. You just need to add this condition:

```yml
- ${{ if eq(variables['isFullMatrix'], false) }}:
  - OSX_x64
```

That will cause to only run this on PRs.